teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -142,10 +147,22 @@ public String getP2pPrivateKeyFile() {
   }
 
   public int getP2pLowerBound() {
+    if (p2pLowerBound < ATTESTATION_SUBNET_COUNT) {
+      LOG.warn(
+          "--p2p-peer-lower-bound set too low. Defaulting to ATTESTATION_SUBNET_COUNT:  {}",
+          ATTESTATION_SUBNET_COUNT);
+      p2pLowerBound = ATTESTATION_SUBNET_COUNT;
+    }
     return p2pLowerBound;
   }
 
   public int getP2pUpperBound() {
+    if (p2pUpperBound < ATTESTATION_SUBNET_COUNT + 10) {
+      LOG.warn(
+          "--p2p-peer-upper-bound set too low. Defaulting to ATTESTATION_SUBNET_COUNT + 10:  {}",
+          ATTESTATION_SUBNET_COUNT + 10);
+      p2pUpperBound = ATTESTATION_SUBNET_COUNT + 10;
+    }
```

Comment:
As discussed, I'd keep the increased defaults but remove this restrictions on the minimums that can be set.

Reply:
Done.